### ISSUE TITLE
Ability to disable dnsmasq

### ISSUE THREAD

**Issue by karbowiak** [subspacecloud/subspace#36]
Hello there - it would be absolutely super if dnsmasq could be disabled.

That's about it, thanks :)

**Comment by CountParadox** [comment]
I agree.. I cant for the life of me even get subspace running because it keeps restarting due to being unable to bind to 53..

**Comment by chase9** [comment]
Checkout #13 for discussion about binding to port :53 on newer systems
